lib/rubocop/cop/method_complexity_patch.rb

```diff
@@ -0,0 +1,18 @@
+# Monkey patching! Here be dragons.
+module RuboCop
+  module Cop
+    module MethodComplexity
+      # RuboCop's default implementation of `add_offense` in `Cop` only gets the
+      # location of the keyword associated with the problem which, for things
+      # like complexity checkers, is just the method def line. This isn't very
+      # useful for checkers where the entire method body is relevant. Fetching
+      # this information from an `Offense` instance is difficult, since the
+      # original AST is no longer available. So it's easier to monkey-path
```

Comment:
`monkey-patch` :star: 

I like the detail in this explanation. If you prefer brevity, what about something like:

`Overwrite RuboCop to send entire offending method as offense, not just def line.`